src/main/resources/db/migration/V23__REFACTOR_USER_ROLES.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE users ADD COLUMN temp_role ENUM('ROLE_ADMIN', 'ROLE_NORMAL') DEFAULT 'ROLE_NORMAL';
```

Comment:
다음처럼 작성하면 한 줄로 변경할 수 있습니다!

```suggestion
ALTER TABLE users MODIFY COLUMN user_role ENUM('ROLE_NORMAL', 'ROLE_ADMIN') DEFAULT 'ROLE_NORMAL';
```

Reply:
그걸 시도해봤는데, 데이터가 이미 들어있을 때 enum 값이다보니 에러가 나더라구요. `ERROR 1265 (01000): Data truncated for column 'user_role' at row 1` 
그래서 임시 칼럼 생성 -> 기존 칼럼 값에 대응된 새로운 칼럼값 생성 -> 기존 칼럼 삭제 -> 새로운 칼럼명 기존 칼럼명으로 변경
절차로 수정하였습니다!

Reply:
근데 이미 생성된 데이터들은 다 로컬 임시 데이터 아닌가요? 굳이 임시 데이터때문에 우회할 필요는 없을 것 같아요! 저희 테스트용 데이터 백업할 때도 데이터 날리고 INSERT문으로 처리할거라서 그냥 데이터 날리고 한 줄로 작성하는 게 좋을 것 같습니다~!